stream-chat-android-client-sample/build.gradle

```diff
@@ -78,6 +78,7 @@ dependencies {
     kapt Dependencies.roomCompiler
 
     // Firebase
+    implementation platform(Dependencies.firebaseBom)
```

Comment:
Using the BoM is the recommended way of keeping Firebase dependencies in sync, as their compatible versions have very different version numbers https://firebase.google.com/docs/android/learn-more#bom